### PR TITLE
Add serverless-apigateway-route-settings

### PR DESCRIPTION
**Pull request by talbotp**
[https://github.com/talbotp/serverless-apigateway-route-settings](https://github.com/talbotp/serverless-apigateway-route-settings)

For configuring [RouteSettings](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apigatewayv2-stage-routesettings.html) on HTTP ApiGatewayv2, currently supports Throttling and Detailed Metrics :slightly_smiling_face:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Add your plugin to [the `plugins.json` file](https://github.com/serverless/commu
 - Do not remove or modify this section. Make all updates to plugins.json -->
 | Plugin | Stats |
 |:---------------------------|:-----------:|
+| **[@Digitalmaas/serverless Plugin Sqs Alarms - `@digitalmaas/serverless-plugin-sqs-alarms`](https://github.com/digitalmaas/serverless-plugin-sqs-alarms)** <br/> by [digitalmaas](http://github.com/digitalmaas) <br/> Simplifies the setup of CloudWatch Alarms to monitor the visible messages in an SQS queue. | ![Github Stars](https://img.shields.io/github/stars/digitalmaas/serverless-plugin-sqs-alarms.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/@digitalmaas/serverless-plugin-sqs-alarms.svg?label=Downloads&style=for-the-badge)|
```
Comment (pgrzesik) [inline README.md:21]: Is there a reason for this being present in your change?

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thank you @talbotp - looks good, I've just noticed one unrelated change being included in your PR - could you clarify that?

File: README.md (hunk 1 above)
Comment (talbotp) [reply README.md:21]: Awesome :slightly_smiling_face: I just ran npm run docs on the master branch and it was added again, looks like it was added to plugins.json but not the README in this [commit] maybe?(https://github.com/serverless/plugins/commit/359202bae3226cb60813d82d30ac2af70f7a8127)?

Comment (pgrzesik) [reply README.md:21]: Thanks, I believe that's the case here :+1:

**Pull request merged**